test/shell/test_scala_import_source_jar.sh

```diff
@@ -22,21 +22,32 @@ test_scala_import_fetch_sources_with_env_bazel_jvm_fetch_sources_set_to() {
 test_scala_import_fetch_sources() {
   local srcjar_name="guava-21.0-src.jar"
   local bazel_out_external_guava_21=$(bazel info output_base)/external/com_google_guava_guava_21_0
+  local expect_failure=$1
 
   set -e
   bazel build //test/src/main/scala/scalarules/test/fetch_sources/...
   set +e
 
-  assert_file_exists $bazel_out_external_guava_21/$srcjar_name
+  assert_file_exists $expect_failure $bazel_out_external_guava_21/$srcjar_name
 }
 
 assert_file_exists() {
-  if [[ -f $1 ]]; then
-    echo "File $1 exists."
-    exit 0
+  if [[ $1 ]]; then
+    if [[ -f $2 ]]; then
+      echo "File $2 exists but we expect no source jars."
+      exit 1
+    else
+      echo "File $2 does not exist."
+      exit 0
+    fi
   else
-    echo "File $1 does not exist."
-    exit 1
+    if [[ -f $2 ]]; then
+      echo "File $2 exists."
+      exit 0
+    else
+      echo "File $2 does not exist."
```

Comment:
can we add "but we expect it not to exist" here to be consistent with line 37?

Reply:
Shouldn't it be "but we expect it to exist" ?

Reply:
yes, I was wrong and made a mistake above.

Reply:
Good idea. Changed.